platforms/gke/base/core/gke_enterprise/configmanagement/git/feature.tf

```diff
@@ -40,10 +43,20 @@ resource "google_gke_hub_feature_membership" "cluster_configmanagement" {
   feature    = google_gke_hub_feature.configmanagement.name
   location   = google_gke_hub_feature.configmanagement.location
   membership = data.google_container_cluster.cluster.name
-  project    = data.google_project.environment.project_id
+  project    = data.google_project.cluster.project_id
 
   configmanagement {
+    management = "MANAGEMENT_AUTOMATIC"
+
     config_sync {
+      enabled       = true
+      source_format = "unstructured"
+
+      git {
+        secret_type = "token"
```

Comment:
Can we use a variable here instead of a hardcoded value in order to support SSH-based authentication as well?